docs/requirements.md

```diff
@@ -2,6 +2,12 @@
 
 This document explains the requirements and recommendations in the underlying databases of ScalarDB to make ScalarDB applications work correctly.
 
+## Common requirements
+
+### Privileges to access the underlying databases
+
+ScalarDB operates the underlying databases not only for CRUD operations but also for creating/altering schema, tables, indexes, and so on. Thus, ScalarDB requires a fully privileged account to access the underlying databases.
```

Comment:
It would be better to add example privilege settings (e.g., GRANT xxx) in each database for EoU. Also, I think we should add the related tests. I will work on it later.

We might need to have a further discussion about whether the full privilege requirement should be relaxed or not since I heard that a user does not want to give it to ScalarDB. One idea (from Takanori) is to note that "grant the full privilege if you use admin feature / API, otherwise grant x, y, z ...", or something like that. But for now, I think the full privilege is the simple and comprehensive solution (with also considering introducing the Auth feature soon).